README.md

```diff
@@ -246,41 +246,17 @@ Create a new migration using the following command:
 npx supabase db diff | npx supabase migration new <migration_name>
 ```
 
-### Password Reset When Forgotten
-If users forgot their password, they can request for a reset. Atomic CRM handle it for you. All the magic is done by the custom route `/forgot-password` and `/set-password` that you can find in the `CRM` component.
```

Comment:
This whole section appeared to be duplicate of `./doc/supabase-configuration.md#login-callback` and `./doc/supabase-configuration.md#customizing-mail-template` so I replaced it with appropriate links